woocommerce/woocommerce
# Update Playwright to 1.46.1

### Changes proposed in this Pull Request:



Updates Playwright from version 1.45.1 to 1.46.1.  The latest version of Playwright includes some enhancements to UI mode and to annotations.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure CI passes
2. Pull branch
3. `pnpm env:restart && pnpm test:e2e`
